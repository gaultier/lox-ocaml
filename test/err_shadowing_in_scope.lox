var a = 1; var a =2; // expect: Forbidden shadowing of variable `a` in the same scope
