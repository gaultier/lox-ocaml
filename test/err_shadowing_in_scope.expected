Forbidden shadowing of variable `a` in the same scope
